content/swagger/resources/families/routes/families_code.yaml

```
@@ -66,3 +66,19 @@ patch:
       $ref: "#/responses/415Error"
     422:
       $ref: "#/responses/422Error"
+
+delete:
+  summary: Delete a family
+  operationId: "delete_families__code__"
+  description: This endpoint allows you to delete a family by code.
+  tags:
+    - Family
+  x-versions:
+    - "SaaS"
+  parameters:
+    - $ref: '#/parameters/code'
+  responses:
+    204:
+      description: Empty content
+    404:
+      $ref: "#/responses/404Error"
```

Not sure if we have to add it but it also could be in case of forbidden or violation
```suggestion
    422:
      $ref: "#/responses/422Error"
    403:
      $ref: "#/responses/403Error"  
```